p2p_wallet/Services/Repositories/BaseRepository/LoadingState.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/// Loading state for a specific time-consuming operation
+enum LoadingState {
+    /// Nothing loaded
+    case initializing
```

Comment:
seems like it is not a process, why not "initialized"? what do you think?

Reply:
Yes, initialized would be better. Thanks